engine/common/hpak.c

```diff
@@ -512,7 +512,7 @@ void HPAK_CheckSize( const char *filename )
 	Q_strncpy( pakname, filename, sizeof( pakname ));
 	COM_ReplaceExtension( pakname, ".hpk" );
 
-	if( FS_FileSize( pakname, false ) > ( maxsize * 1000000 ))
+	if( FS_FileSize( pakname, false ) > ( maxsize * 1048576 ))
```

Comment:
What's this magical number? 1024 * 1024?

Reply:
Yes, it's 1024 * 1024, 1 megabyte